packages/react/src/hooks/asset/useCreateAsset.test.ts

```diff
@@ -63,7 +62,9 @@ describe('useCreateAsset', () => {
           "mutate": [Function],
           "mutateAsync": [Function],
           "status": "success",
-          "uploadProgress": undefined,
+          "uploadProgress": [
```

Comment:
I think we can handle this upload progress better. Some devs will want to show individual upload progress (and this is great), and some will take this array of values and average them to get the current upload progress as a singular value. Could we find a way to provide both?

Maybe `uploadProgress = { average: 0.8, files: [0.7, 0.9] }`? WDYT?

Reply:
Thanks, that is a great idea. I will add the average too.